Makefile

```diff
@@ -46,3 +45,38 @@ python-bootstrap: venv
 
 venv:
 	test -d venv || virtualenv venv
+
+memorydiskserver-docker-image:
```

Comment:
I've attempted to address this in https://github.com/ClusterHQ/dvol/blob/docker-interacting-dvol-tests-FLOC-4258/scripts/build-golang-docker-image.sh